lib/brains/forms.py

```diff
@@ -1,5 +1,40 @@
 from django import forms
+from django.core.exceptions import ObjectDoesNotExist
+
+from lib.buyers.models import Buyer
 
 
 class BuyerForm(forms.Form):
     uuid = forms.CharField(max_length=255)
+
+
+class PaymentMethodForm(forms.Form):
+    buyer_uuid = forms.CharField(max_length=255)
+    nonce = forms.CharField(max_length=255)
+
+    def clean_buyer_uuid(self):
+        data = self.cleaned_data
+
+        try:
+            self.buyer = Buyer.objects.get(uuid=data['buyer_uuid'])
+        except ObjectDoesNotExist:
+            raise forms.ValidationError('Buyer does not exist.')
```

Comment:
One example. No matching string for SELLER_UUID not being unique or any other errors on that form. 

https://github.com/mozilla/solitude/search?utf8=%E2%9C%93&q=BUYER_UUID_ALREADY_EXISTS&type=Code